cove/cove_360/templates/cove_360/input.html

```diff
@@ -0,0 +1,42 @@
+{% extends "input.html" %}
+
+{% block precontent %}
+<div class="row">
+  <div class="col-md-8">
+    <h2>Check your data</h2>
+    <p>Upload or provide a link to a file, or paste JSON to check and convert your data and get feedback on whether it is valid 360Giving data.</p>
+  </div>
+</div>
+{% endblock %}
+
+{% block postcontent %}
+{% if DATA_SUBMISSION_ENABLED %}
+<div class="row">
+    <div class="col-md-8">
+      <h2>Submit your data</h2>
+      <p>Submit your 360Giving data file to the <a href="https://data.threesixtygiving.org/">360Giving Data Registry</a>. Your file will also be checked to make sure you are submitting valid 360Giving data.</p>
+
+    </div>
+  </div>
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <form method="GET" action="." id="self-publishing-form">
+      {% csrf_token %}
+      <input type="hidden" name="self_publishing" value="true">
+      <div class="form-group">
+        <label class="control-label">Provide a link to your file:</label>
+        <input type="url" name="source_url" class="form-control">
```

Comment:
```suggestion
        <label for="source-url-input" class="control-label">Provide a link to your file:</label>
        <input id="source-url-input" type="url" name="source_url" class="form-control">
```
Linking the label and input with `for` makes the form more accessible.